dbt_schema_builder/builder.py

```diff
@@ -10,7 +10,8 @@
 import yaml
 from dbt.config import RuntimeConfig
 from dbt.exceptions import DatabaseException
-from dbt.logger import GLOBAL_LOGGER as logger
+from dbt.events import AdapterLogger
+logger = AdapterLogger("Snowflake")
```

Comment:
nit: can you move this line down below? Keeping imports separate from actual code helps readability.